en/api/clapi.md

```diff
@@ -3569,18 +3569,26 @@ Columns are the following:
 
 In order to add an instance you use the **ADD** action:
```

Comment:
I think we should avoid document SSH Poller in latest version. It will be dropped later and might lead to unwanted mistake.